cypress/e2e/functional/opal/paymentTerms/PO-471_collectionOrder_adultOrYouth.feature

```diff
@@ -0,0 +1,131 @@
+        And I see "For example, 31/01/2023" hint text above the "Date of collection order" date picker
+        And I enter "20/05/2024" into the "Date of collection order" date field
+        When I select the "Pay in full" radio button
+        And I see "For example, 31/01/2023" hint text above the "Enter pay by date" date picker
+        And I enter "12/09/2023" into the "Enter pay by date" date field
+
+        Examples:
+            | dateOfBirth |
+            | 20/03/2000  |
+
+    Scenario Outline: AC-1a, AC-3 positive: the 'Has a collection order been made?' field should be presented only, if the defendant is an Adult
+        And I enter "20/03/2000" into the Date of birth field
+        And I click the "Return to account details" button
+        Then I see the status of "Personal details" is "Provided"
+        And I click on the "Payment terms" link
+        Then I see "Payment terms" on the page header
+
+        And I select the "No" radio button under the "Has a collection order been made?" section
+        And I select the "Make collection order today" checkbox
+        When I select the "Pay in full" radio button
+        And I see "For example, 31/01/2023" hint text above the "Enter pay by date" date picker
+        And I enter "12/09/2023" into the "Enter pay by date" date field
+
+    Scenario Outline: AC4 -negative: If the user has not selected a value for the 'Has a collection order been made?' field and clicks either the 'Return to account details' or 'Add account comments and notes' buttons
+        And I click the "Return to account details" button
+        Then I see the status of "Personal details" is "Provided"
+        And I click on the "Payment terms" link
+        Then I see "Payment terms" on the page header
+        When I click the "<returnButton>" button
+        Then I see the error message "Select whether there was a collection order" at the top of the page
+
+        Examples:
+            | returnButton                   |
+            | Return to account details      |
+            | Add account comments and notes |
+
+    Scenario Outline: AC5 -negative: If the user has not selected a value for the 'Has a collection order been made?' field and clicks either the 'Return to account details' or 'Add account comments and notes' buttons
+        And I click the "Return to account details" button
+        Then I see the status of "Personal details" is "Provided"
+        And I click on the "Payment terms" link
+
+        And I select the "Yes" radio button under the "Has a collection order been made?" section
+        Then I see "Payment terms" on the page header
+        When I click the "<returnButton>" button
+        Then I see the error message "Enter date collection order made" at the top of the page
+
+        Examples:
+            | returnButton                   |
+            | Return to account details      |
+            | Add account comments and notes |
```

Comment:
same comment as above

Reply:
No Problem at all. Same steps I could repeat in the same scenario. In this way we could avoid 'Background' steps twice running